MonoGame.Framework/Input/Keyboard.Windows.cs

```diff
@@ -4,21 +4,57 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.Xna.Framework.Input
 {
     public static partial class Keyboard
     {
-        static List<Keys> _keys;
+        private static readonly Keys[] DefinedKeys;
+
+        private static readonly byte[] _keyState = new byte[256];
```

Comment:
Yeah, that's a fair point. I was going for better performance at the cost of a bit more memory usage, but didn't really stop to think if I could do it using less memory. Leave it with me, I'll improve it.